packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -200,7 +199,7 @@ function NetworkContainer({
   }, [customAddress, walletAddressLowercased])
 
   return (
-    <>
+    <div>
```

Comment:
This gotta be a div, if it's a fragment then a gap from the parent wrapper is applied between the custom destination banner and the network component.